featuretools/tests/dfs_tests/test_deep_feature_synthesis.py

```diff
@@ -360,13 +351,13 @@ def test_seed_features_added_with_identity_features(es):
 
 
 def test_dfs_builds_on_seed_features_more_than_max_depth(es):
-    seed_feature_sessions = Count(es['log']["id"], es['sessions']) > 2
-    seed_feature_log = Hour(es['log']['datetime'])
-    session_agg = Last(seed_feature_log, es['sessions'])
+    seed_feature_sessions = ft.Feature(es['log']["id"], parent_entity=es['sessions'], primitive=Count)
```

Comment:
I don't think it changes the test but `seed_feature_sessions` is missing the `> 2`

Reply:
ya, i noticed that. i dont think adding the `>2` improved the test so i simplified it